components/UserForm.js

```diff
@@ -1,6 +1,8 @@
+import styled from "styled-components";
+
 export default function UserForm({ client, handleSubmit, handleInputChange }) {
   return (
-    <form onSubmit={handleSubmit}>
+    <StyledForm onSubmit={handleSubmit}>
```

Comment:
It's convention to name a function "on..." (here onSubmit) until you actually handle the Submit, so always pass it as onSubmit until the component or page where you actually declare the function handleSubmit
```suggestion
    <StyledForm onSubmit={onSubmit}>
```